Dockerfile

```
@@ -21,7 +19,7 @@ RUN	CGO_ENABLED=0 GOOS=linux go build -mod vendor -ldflags \
     "-w -s -X sigs.k8s.io/aws-encryption-provider/pkg/version.Version=$TAG" \
     -o bin/aws-encryption-provider cmd/server/main.go
 
-FROM busybox AS aws-encryption-provider
-COPY --from=build /etc/ssl/certs/ /etc/ssl/certs/
+FROM public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base:latest
```

Same here on parameterization
```suggestion
ARG BASE_IMAGE=public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base:latest
FROM ${BASE_IMAGE}
```